src/app/modules/shared-components/components/input/input.component.ts

```diff
@@ -23,11 +23,17 @@ export class InputComponent {
   @Input() buttonIcon?: string; // a font-awesome icon identifier for the input button
   @Input() hasClearButton = false;
 
+  @Input() pTooltip = '';
+  @Input() tooltipEvent = 'hover';
+  @Input() tooltipPosition = 'right';
+  @Input() tooltipDisabled = false;
```

Comment:
don't you have to disable it by default to prevent having it enabled everywhere?

Reply:
The tooltip is enabled only if there is content in the `pTooltip `. `tooltipDisabled` is only doing something if the tooltip is shown. I would argue that both setting it to true or false by default can make sense, but in primeng its default value is false so I did the same